flink-formats/flink-hadoop-bulk/archunit-violations/db4de53e-d09e-4fb0-bdbc-429c1b64686f

```diff
@@ -0,0 +1,12 @@
+org.apache.flink.formats.hadoop.bulk.committer.HadoopRenameCommitterHDFSITCase does not satisfy: only one of the following predicates match:\
```

Comment:
Why is this added? It looks like there is some problem with the archunit tests and subclasses? :thinking:

Reply:
I've removed these exceptions -- it looks like this was a consequence of having the (require) `MiniClusterExtension` being present through the abstract base class, instead of the direct implementation.